test/chat/dialog_test.exs

```diff
@@ -110,7 +110,7 @@ defmodule Chat.Dialogs.DialogTest do
 
     [bob_version] = Dialogs.read(dialog, bob)
 
-    assert :file = bob_version.type
+    assert :audio = bob_version.type
```

Comment:
why `:file` becomes `:audio`?

Reply:
Before, we didn't support audio files so it was recognized as `:file`, but now we do and the test uses a fake file with `audio/mp3` mime type:
https://github.com/Buckitup-chat/chat/blob/6595e1f5110f5fcae49d0a4a9eceb2189c1a32a2/test/support/fake_data.ex#L5-L11